static/js/app.js

```diff
@@ -235,6 +235,15 @@ function addEventListeners() {
     });
   }
 
+  const tipsCard = document.querySelector(".js-tips");
```

Comment:
Question: Should there be some sort of logic to show/hide this panel? Currently there's the `is-hidden` class on page load and at no point does it get removed/toggled.

Reply:
It gets removed here: https://github.com/mozilla/fx-private-relay/blob/2fa366fd6144d699a261ed8521b47e2cb73f6264/static/js/app.js#L241

(And it's not possible at this time to manually re-show it.)